docs/data/charts/line-demo/LiveLineChart.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { LineChart } from '@mui/x-charts/LineChart';
+
+const xLabels = [
+  'Page A',
+  'Page B',
+  'Page C',
+  'Page D',
+  'Page E',
+  'Page F',
+  'Page G',
+];
+
+const randBetween = (min: number, max: number) => {
+  return Math.floor(Math.random() * (max - min + 1) + min);
+};
+
+export default function LiveLineChart() {
```

Comment:
Added this demo because I suppose live data is common among charts and I wanted to test how the tooltip reacted to changing data.